src/global_styling/variables/_typography.scss

```
@@ -75,3 +75,6 @@ $euiTitles: (
     'letter-spacing': -.03em,
   ),
 ) !default;
+
+// TODO: Moved from EuiText - can be removed once all usages are converted to Emotion
+$euiTextConstrainedMaxWidth: #{'max(64ch, 75%)'};
```

Yeah, this change is having ill-effects on EuiEmptyPrompt. I think it's safe to remove it all together and hard-code the old value in the empty prompt Sass.

```suggestion
$euiTextConstrainedMaxWidth: #{'max(64ch, 75%)'};
```

---

https://github.com/elastic/eui/pull/5895/commits/8a4ed40836c8e2f4a65ec13e56c795da6bb9b026